Pear/Models/User.swift

```diff
@@ -8,6 +8,24 @@
 
 import Foundation
 
+struct LoginUser: Codable {
+
+    let didOnboard: Bool
+    let graduationYear: String?
+    let groups: [String]
+    let hometown: String?
+    let facebook: String?
+    let firstName: String
+    let interests: [String]
+    let instagram: String?
+    let lastName: String
+    let major: String
+    let netID: String
+    let profilePictureURL: String?
+    let pronouns: String?
+    
```

Comment:
why are pronouns, hometown, and grad year optional if the user has to input them to onboard but major, name, groups, and interests are not?

Reply:
Just how backend structured this...